include/clap/ext/draft/undo.h

```diff
@@ -1,8 +1,11 @@
 #pragma once
 
 #include "../../plugin.h"
+#include "../../stream.h"
 
-static CLAP_CONSTEXPR const char CLAP_EXT_UNDO[] = "clap.undo/2";
+static CLAP_CONSTEXPR const char CLAP_EXT_UNDO[] = "clap.undo/4";
+static CLAP_CONSTEXPR const char CLAP_EXT_UNDO_CONTEXT[] = "clap.undo_context/4";
+static CLAP_CONSTEXPR const char CLAP_EXT_UNDO_DELTA[] = "clap.undo_delta/4";
```

Comment:
do you have the "/4" here to keep them in sync with the CLAP_EXT_UNDO ?

Reply:
We did some internal iteration with the /3 and I bumped it so we don't get confused with ABI mismatch. Also even if /3 wasn't released, there was still a /3 to /4 iteration internally.

Reply:
Ah, and yes I kept the same number in all of them so they are in sync.